Formula/minimal-racket.rb

```diff
@@ -2,16 +2,16 @@ class MinimalRacket < Formula
   desc "Modern programming language in the Lisp/Scheme family"
   homepage "https://racket-lang.org/"
   url "https://mirror.racket-lang.org/installers/7.9/racket-minimal-7.9-src-builtpkgs.tgz"
-  sha256 "293aa8ef709a6240472f16833351ba66a9e461261d2813c7fb1cc5ddf59c3000"
+  sha256 "85b201aebc1ad1ec98ac590e18052d7ef8a81af280244d00ca1c28e8543b3fe9"
   license any_of: ["MIT", "Apache-2.0"]
+  revision 1
 
   livecheck do
     url "https://download.racket-lang.org/all-versions.html"
     regex(/>Version ([\d.]+)/i)
   end
 
   bottle do
-    cellar :any
```

Comment:
I don't think you're supposed to edit the bottle block.

Reply:
reverted.